.travis.yml

```diff
@@ -26,6 +26,13 @@ matrix:
            ALT=i686-unknown-linux-gnu
       rust: beta
 
+    - env: TARGET=x86_64-unknown-linux-gnu
+           ALT=i686-unknown-linux-gnu
+      rust: nightly
```

Comment:
I think this should list an explicit Rust version, right? (the minimum supported rust version?)

Reply:
For projects that have a msrv. But as you pointed out cargo doesn't. Maybe stable?

Reply:
Oh sure yeah Cargo *technically* doesn't but if we want to emulate "best practices" here then I think we should probably attempt it here as well. For now I'd just pick 1.27 as the current stable